### PR TITLE
feat: add default template and remove the need for a custom default key

### PR DESCRIPTION
**Pull request by Nebulis**
image

### CODE REVIEW

File: example/decentralized-renderer/templates/govtechDemoCert/common/SimplePrivacyFilterBanner.tsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  justify-content: space-between;
+  padding: 20px;
+  .icon {
+    cursor: pointer;
+  }
+`;
+/**
+ * Banner with icon to toggle certificate obfuscation mode
+ */
+export const SimplePrivacyFilterBanner: FunctionComponent<SimplePrivacyFilterBannerProps> = ({
```
Comment (yehjxraymond) [inline example/decentralized-renderer/templates/govtechDemoCert/common/SimplePrivacyFilterBanner.tsx]: Wondering if this name is still appropriate for this component and file. Maybe something like `WarningBanner`?

Comment (Nebulis) [reply example/decentralized-renderer/templates/govtechDemoCert/common/SimplePrivacyFilterBanner.tsx]: this component is only for the example provided, it's the SimplePrivacyFilterBanner which is used in our demo certificate and was reused there. I didnt notice when I brought it here

Comment (yehjxraymond) [reply example/decentralized-renderer/templates/govtechDemoCert/common/SimplePrivacyFilterBanner.tsx]: Yea i got that, but do we want to give it a better name? People will be less confused that way. Can imagine people trying to figure how does it do privacy filtering lol

Comment (Nebulis) [reply example/decentralized-renderer/templates/govtechDemoCert/common/SimplePrivacyFilterBanner.tsx]: Ok will rename to banner

Comment (Nebulis) [reply example/decentralized-renderer/templates/govtechDemoCert/common/SimplePrivacyFilterBanner.tsx]: Done

**Pull request merged**